<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusFileRun</class>
 <widget class="QDialog" name="MotusFileRun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="fileDataButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>21</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>文件运动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFileButton">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>21</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePathLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>90</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="filePathLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>92</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>372</y>
     <width>431</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="zongQinLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>191</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纵倾</string>
   </property>
  </widget>
  <widget class="QLabel" name="hengYaoLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>221</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横摇</string>
   </property>
  </widget>
  <widget class="QLabel" name="hangXiangLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>251</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航向</string>
   </property>
  </widget>
  <widget class="QLabel" name="hengYiLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>281</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横移</string>
   </property>
  </widget>
  <widget class="QLabel" name="qianChongLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>311</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>前冲</string>
   </property>
  </widget>
  <widget class="QLabel" name="shengJiangLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>341</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升降</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxValueLabel">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>168</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>最大值°/mm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxZongQinLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>189</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHengYaoLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>219</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHangXiangLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>249</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHengYiLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>279</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxQianChongLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>309</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxShengJiangLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>339</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minShengJiangLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>339</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minValueLabel">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>168</y>
     <width>78</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>最小值°/mm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minHangXiangLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>249</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minZongQinLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>189</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minHengYiLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>279</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minHengYaoLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>219</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minQianChongLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>309</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxQianChongAccLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>309</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="maxSpeedLabel">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>167</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>最大速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHangXiangSpeedLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>249</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxZongQinSpeedLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>189</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHengYiSpeedLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>279</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxZongQinAccLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>189</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHengYiAccLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>279</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHengYaoSpeedLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>219</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxShengJiangAccLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>339</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxShengJiangSpeedLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>339</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="maxAccLabel">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>167</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>最大加速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHangXiangAccLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>249</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHengYaoAccLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>219</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxQianChongSpeedLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>309</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataLenghtLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="runTimeLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>131</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>运行时长</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="runTimeLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>131</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dataLengthLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数据长度</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedremarklabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>最大加速度：纵倾.横摇.航向单位为度每平方秒,横移.前冲.升降单位为毫米平方秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="posremarklabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>最大速度：纵倾.横摇.航向单位为度每秒,横移.前冲.升降单位为毫米每秒</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
